lib/mayaUsd/resources/scripts/mayaUsdMergeToUSDOptions.py

```diff
@@ -255,28 +255,28 @@ def getDefaultMergeToUSDOptionsDict():
     Retrieves the default merge-to-USD options.
     """
     return {
```

Comment:
Making the default have real value provides better validation of input values.

Reply:
I just have one question before merging? Does that have any impact on users, such as are any of these values corresponding to optionVar's and now by changing their value the users saved values won't work? Or are these values used in any MEL scripting command and if a user has a MEL script using that command it will fail now since these values are no longer strings?

Reply:
No, the values are saved all together in one long option var string. The purpose of giving them now their right type is for the validation of input and conversion from string to dictionary (either Python or USD VtDictionary).

Reply:
The conversion using type is done in C++ and in mayaUsdOptions.py in the function _convertTextToType()

Reply:
Ahah! Of course, the options in array form did not want to be converted to an array... These things are not obvious...